sunpy-core.md

```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
```

Comment:
2.0 is due to be released in May 2020 btw...

Reply:
I see these as goals? if it comes along and we don't have this implemented it moves to the next release.

Reply:
Then why bother splitting by release at all? It would be better to have a semi-prioritised list?

Reply:
Well that is a good point but we should have a place to let people know what are the plans for each release.

Reply:
This is why I don't think PRs to a markdown doc is a good way to do this going forwards.